### PR TITLE
Changes to squad color code

### PR DESCRIPTION
**Pull request by BeagleGaming1**
# About the pull request

Changes the squad text and equipment colors from a list into a variable that can be edited

# Explain why it's good for the game

This allows the squad information to be edited much easier mid-round, allowing the creation of new squads with new colors, and edits to pre-existing squads.

# Changelog
:cl:
code: Changed squad color code from a list to a variable
/:cl:

### CODE REVIEW

**Comment by BeagleGaming1** [comment]
I have probably broken so much with this PR. It worked locally but I don't know how it will go with more players

File: code/modules/clothing/gloves/marine_gloves.dm [hunk 1]
```diff
@@ -22,13 +22,27 @@
...
+	glove_overlay.color = current_human.assigned_squad.equipment_color
+	ret.overlays += glove_overlay
 	return ret
 
+/*
+	var/image/ret = ..()
+	if(adopts_squad_color && slot == WEAR_HANDS && istype(current_human) && current_human.assigned_squad)
+		ret.overlays += current_human.assigned_squad.equipment_color
+	return ret
+*/
```
Comment (harryob) [inline code/modules/clothing/gloves/marine_gloves.dm:45]: commented code

File: code/game/jobs/job/marine/squads.dm [hunk 2]
```diff
@@ -96,35 +118,40 @@
 
 /datum/squad/marine/alpha
 	name = SQUAD_MARINE_1
-	color = 1
+	equipment_color = "#e61919"
+	chat_color = "#e67d7d"
```
Comment (harryob) [inline code/game/jobs/job/marine/squads.dm:122]: these might wanna be defines like the minimap colors for visibility idfk tho

**Review by harryob — CHANGES_REQUESTED** [review]

**Review by harryob — APPROVED** [review]

**Comment by BeagleGaming1** [comment]
Found some bugs from the test merge:
- If squad name is changed, req supply pads break
- If squad name is changed, crew manifest starts to runtime

**Pull request merged**